native/src/main/java/io/ballerina/stdlib/websubhub/ModuleUtils.java

```diff
@@ -37,4 +42,39 @@ public static void setModule(Environment environment) {
     public static Module getModule() {
         return module;
     }
+
+    public static void notifySuccess(CompletableFuture<Object> future, Object result) {
+        if (result instanceof BError) {
+            BError error = (BError) result;
+            if (!isModuleDefinedError(error)) {
+                error.printStackTrace();
+            }
+        }
+
```

Comment:
Lets remove this new line

Reply:
Fixed with 83c65f98306e0da9824443382f65f035661ea530